### ISSUE TITLE
unable to alias bytes

### ISSUE THREAD

**Issue by gotwarlost** [ardielle/ardielle-tools#81]
```
$ cat x.rdl
type Content Bytes;

type Foo Struct {
	Content Bar;
}

resource Foo GET "/" {
}
$ rdl -ps generate go-model x.rdl
$ go build
./x_schema.go:16: undefined: rdl.NewBytesTypeBuilder
```

**Comment by gotwarlost** [comment]
what I really want to do with the alias is to enforce a `maxsize` on it.

**Comment by boynton** [comment]
Fixed in ardielle-go, change 7222aadb86c55395a52e006c27705ad09d734f0d
